packages/simple-diff/simple-diff.0.1/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Gabriel Jaldon <gjaldon85@gmail.com>"
+authors: "Gabriel Jaldon <gjaldon85@gmail.com>"
+homepage: "https://github.com/gjaldon/simple_diff"
+bug-reports: "https://github.com/gjaldon/simple_diff/issues"
+license: "MIT"
+dev-repo: "https://github.com/gjaldon/simple_diff"
```

Comment:
should be either `git://github.com/gjaldon/simple_diff` or `https://github.com/gjaldon/simple_diff.git` (I prefer the latter due to https)